homeassistant/components/philips_js/const.py

```diff
@@ -2,6 +2,7 @@
 
 DOMAIN = "philips_js"
 CONF_SYSTEM = "system"
+CONF_ALLOWNOTIFY = "allownotify"
```

Comment:
Please separate words with underscore in variable names.
```suggestion
CONF_ALLOW_NOTIFY = "allow_notify"
```